azure/iac/master-compliance-test.json

```diff
@@ -2621,7 +2621,7 @@
                             "id": "PR-AZR-0113-ARM",
                             "eval": "data.rule.fileService",
                             "message": "data.rule.fileService_err",
-                            "remediationDescription": "",
+                            "remediationDescription": "Make sure StorageAccount has ( properties.encryption.services with [ name='file' and enabled=true ] )",
```

Comment:
**See my validation message at line 2496 (for id PR-AZR-0109-ARM).** make the description format like this one.